app/logic/apiEndpoint.py

```diff
@@ -0,0 +1,48 @@
+from flask import jsonify
+from datetime import datetime
+from app.config.loadConfig import*
+
+from app.models.laborStatusForm import LaborStatusForm
+from app.models.department import Department
+from app.models.formHistory import FormHistory
+from app.models.term import Term
+
+def getLaborInformation(orgCode = "", bNumber = ""):
+
+    if not orgCode:
+        # The default right now is just to search within CELTS Departments, so setting it 
+        # here if it is not passed in. 
+        orgCode = 2084

```

Comment:
Consider setting this orgCode default as the default parameter in the function definition. Or, you might consider making this a config constant if we plan on changing this at all or not if this is temporary.